proto/ibc/applications/fee/v1/query.proto

```diff
@@ -53,3 +59,20 @@ message QueryIncentivizedPacketResponse {
   // Incentivized_packet
   ibc.applications.fee.v1.IdentifiedPacketFee incentivized_packet = 1;
 }
+
+// QueryIncentivizedPacketsForChannelRequest is the request type for querying for all incentivized packets
+// for a specific channel
+message QueryIncentivizedPacketsForChannelRequest {
+  // pagination defines an optional pagination for the request.
+  cosmos.base.query.v1beta1.PageRequest pagination = 1;
```

Comment:
I'd like to clean up the protodoc comments to document each field similarly to this one. Just a concise lowercase comment for each field in the proto message. 

I can open an issue and we can address it after this is merged!